environment.yaml

```diff
@@ -5,7 +5,7 @@ channels:
 dependencies:
   - imageio
   - pandas
-  - python=3.8
+  - python=3.6
```

Comment:
Plz, revert back to `3.8`.

Reply:
Env.yaml is changed to python 3.8. color_smoother.py adds EOL.